SU-SWS/saa_victoria_subtheme
# Adapt 476  single card font sizes

# READY FOR REVIEW

# Summary
- TL;DR - increase font size of h2 and paragrpah in a single icon card in a single card row

# Review By (Date)
- 7/21
# Urgency
- needed today!

# Steps to Test

1. pull down
1. clear cache
2. test sizes of single-icon card in a row, see h2 and paragraph is bigger 

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
